app/controllers/rides_controller.rb

```diff
@@ -0,0 +1,6 @@
+class RidesController < ApplicationController
+
+  def show
+  end
```

Comment:
Please remove whitespace above and below. While you're at it, you can make this one line as so:
```
def show; end
```